src/main/kotlin/no/nav/syfo/service/VarselBusService.kt

```diff
@@ -1,24 +1,35 @@
 package no.nav.syfo.service
 
-import no.nav.syfo.kafka.consumers.varselbus.domain.ArbeidstakerHendelse
-import no.nav.syfo.kafka.consumers.varselbus.domain.EsyfovarselHendelse
+import no.nav.syfo.kafka.consumers.varselbus.domain.*
 import no.nav.syfo.kafka.consumers.varselbus.domain.HendelseType.*
-import no.nav.syfo.kafka.consumers.varselbus.domain.NarmesteLederHendelse
-import org.slf4j.Logger
-import org.slf4j.LoggerFactory
+import org.slf4j.*
 
 class VarselBusService(
     val motebehovVarselService: MotebehovVarselService,
-    val oppfolgingsplanVarselService: OppfolgingsplanVarselService
+    val oppfolgingsplanVarselService: OppfolgingsplanVarselService,
+    val dialogmoteStatusVarselService: DialogmoteStatusVarselService
 ) {
     private val log: Logger = LoggerFactory.getLogger("no.nav.syfo.service.VarselBusService")
     fun processVarselHendelse(varselHendelse: EsyfovarselHendelse) {
         log.info("Behandler varsel av type ${varselHendelse.type}")
         when (varselHendelse.type) {
             NL_OPPFOLGINGSPLAN_OPPRETTET,
             NL_OPPFOLGINGSPLAN_SENDT_TIL_GODKJENNING -> oppfolgingsplanVarselService.sendVarselTilNarmesteLeder(varselHendelse.toNarmestelederHendelse())
+
             NL_DIALOGMOTE_SVAR_MOTEBEHOV -> motebehovVarselService.sendVarselTilNarmesteLeder(varselHendelse.toNarmestelederHendelse())
             SM_DIALOGMOTE_SVAR_MOTEBEHOV -> motebehovVarselService.sendVarselTilArbeidstaker(varselHendelse.toArbeidstakerHendelse())
+
+            NL_DIALOGMOTE_INNKALT -> dialogmoteStatusVarselService.sendVarselTilNarmesteLeder(varselHendelse.toNarmestelederHendelse())
+            SM_DIALOGMOTE_INNKALT -> dialogmoteStatusVarselService.sendVarselTilArbeidstaker(varselHendelse.toArbeidstakerHendelse())
+            NL_DIALOGMOTE_AVLYST -> dialogmoteStatusVarselService.sendVarselTilNarmesteLeder(varselHendelse.toNarmestelederHendelse())
+            SM_DIALOGMOTE_AVLYST -> dialogmoteStatusVarselService.sendVarselTilArbeidstaker(varselHendelse.toArbeidstakerHendelse())
+            NL_DIALOGMOTE_REFERAT -> dialogmoteStatusVarselService.sendVarselTilNarmesteLeder(varselHendelse.toNarmestelederHendelse())
+            SM_DIALOGMOTE_REFERAT -> dialogmoteStatusVarselService.sendVarselTilArbeidstaker(varselHendelse.toArbeidstakerHendelse())
+            NL_DIALOGMOTE_NYTT_TID_STED -> dialogmoteStatusVarselService.sendVarselTilNarmesteLeder(varselHendelse.toNarmestelederHendelse())
+            SM_DIALOGMOTE_NYTT_TID_STED -> dialogmoteStatusVarselService.sendVarselTilArbeidstaker(varselHendelse.toArbeidstakerHendelse())
```

Comment:
Jeg tror du kan slå sammen de som kaller samme metode, slik som på linje 16-17

```suggestion
            NL_DIALOGMOTE_INNKALT, NL_DIALOGMOTE_AVLYST, NL_DIALOGMOTE_REFERAT, NL_DIALOGMOTE_NYTT_TID_STED -> 
            dialogmoteStatusVarselService.sendVarselTilNarmesteLeder(varselHendelse.toNarmestelederHendelse())
            SM_DIALOGMOTE_INNKALT, SM_DIALOGMOTE_AVLYST, SM_DIALOGMOTE_REFERAT, SM_DIALOGMOTE_NYTT_TID_STED -> 
            dialogmoteStatusVarselService.sendVarselTilArbeidstaker(varselHendelse.toArbeidstakerHendelse())
```